llvm/test/Instrumentation/AllocToken/basic.ll

```diff
@@ -0,0 +1,84 @@
+; RUN: opt < %s -passes=inferattrs,alloc-token -alloc-token-mode=increment -S | FileCheck %s
```

Comment:
--implicit-check-not="call ptr @_Znwm" is better guaranty that it does not miss one.

However, how bad is auto-generated test?
Less fork for future maintainers.

Reply:
I would even precommit them after attribute PR and before this one, 
so it's easy to see effect of transformation as a DIFF